src/logendpoint.h

```diff
@@ -44,6 +44,7 @@ struct LogOptions {
     MavDialect mavlink_dialect{MavDialect::Auto}; // conf "MavlinkDialect"
     unsigned long min_free_space;                 // conf "MinFreeSpace"
     unsigned long max_log_files;                  // conf "MaxLogFiles"
+    int fcu_id{-1};                                   // conf "LogMasterId"
```

Comment:
LogMasterId?

Reply:
leftover from some thinking on the name...